src/MeadeCommandProcessor.cpp

```diff
@@ -816,22 +816,30 @@ bool gpsAqcuisitionComplete(int &indicator);  // defined in c72_menuHA_GPS.hpp
 //      Returns:
 //        "HHMMSS"
 //
-// :XGh#
+// :XGhN#
 //      Description:
-//        Get Heater values
+//        Get value for heaters
 //      Information:
-//        Get the current values for both heaters
+//        Get value for heater N in the range from 0-10, or both heaters if no N is given.
 //      Returns:
-//        "NNN,NNN"
+//        "NNN,NNN#" or "N#"
```

Comment:
`"NN,NN# or "NN# where NN is the power the heaters are set to in the range 0 (off) to 10 (full power).`